backend/packages/Upgrade/src/api/controllers/ExperimentClientController.v6.ts

```diff
@@ -177,16 +173,18 @@ export class ExperimentClientController {
     @Req()
     request: AppRequest,
     @Body({ validate: true })
-    experimentUser: ExperimentUserValidator
+    experimentUser: ExperimentUserValidatorv6
   ): Promise<Pick<ExperimentUser, 'id' | 'group' | 'workingGroup'>> {
     request.logger.info({ message: 'Starting the init call for user' });
     // getOriginalUserDoc call for alias
-    const experimentUserDoc = await this.experimentUserService.getUserDoc(experimentUser.id, request.logger);
+    const experimentUserDoc = request.userDoc;
     // if reinit call is made with any of the below fields not included in the call,
     // then we will fetch the stored values of the field and return them in the response
     // for consistent init response with 3 fields ['userId', 'group', 'workingGroup']
```

Comment:
Can we do 
`    const { id = request.get('User-Id'), group, workingGroup } = { ...experimentUserDoc, ...experimentUser };
`